test/e2e/e2e_test.go

```diff
@@ -477,7 +477,7 @@ var _ = Describe("controller", Ordered, func() {
 				}
 
 				for resource := range nodeResult.Status.Capacity {
-					if strings.HasPrefix(resource, "org.instaslice/") {
+					if strings.HasPrefix(resource, orgInstaslicePrefix) {
```

Comment:
The constant `orgInstaslicePrefix` has to be defined in this package as a constant.